### PR TITLE
Couchbase Integration

### PR DESCRIPTION
**Pull request by David00154**
I have implemented the new integration for Couchbase. After some initial testing, it looks to be working as expected. Please do let me know if there is anything that I have missed or if there are any bugs.
N.B You should use N1QL which is the query language for couchbase, in order to make your queries.

### CODE REVIEW

**Comment by ZoranPandovski** [comment]
recheckcla

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb_native/blob/stable/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   ***I have read the CLA Document and I hereby sign the CLA***
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

**Comment by David00154** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by ZoranPandovski** [comment]
recheckcla

**Comment by David00154** [comment]
> recheckcla

How do I recheck cla???

**Comment by ZoranPandovski** [comment]
@David00154 This is for the automated bot :smile: It looks like there is a bug with the GH action we are using https://github.com/contributor-assistant/github-action/issues/108

**Comment by David00154** [comment]
please how do I recheck cla??

**Comment by David00154** [comment]
> @David00154 This is for the automated bot smile It looks like there is a bug with the GH action we are using [contributor-assistant/github-action#108](https://github.com/contributor-assistant/github-action/issues/108)

So what do I do??

**Comment by ZoranPandovski** [comment]
No worries, we will solve this :)

**Comment by David00154** [comment]
ok

**Comment by ZoranPandovski** [comment]
recheckcla

**Comment by David00154** [comment]
Seems like the pr haven't been reviewed 😢

File: mindsdb/integrations/handlers/couchbase_handler/couchbase_handler.py [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+            data = {}
+            # keys = []
+            for collection in row_iter:
+                # data.append()
+                # print(type(collection))
+                for collection_name, row in collection.items():
+                    if isinstance(row, dict):
+                        for k, v in row.items():
+                            if data.get(k) is None:
+                                data[k] = data.get(k) or []
```
Comment (StpMax) [inline mindsdb/integrations/handlers/couchbase_handler/couchbase_handler.py:147]: better to do just
```
if data.get(k) is None:
    data[k] = []
```

**Review by StpMax — APPROVED** [review]
Except few unnecessary `print` and comments all looks fine

File: mindsdb/integrations/handlers/couchbase_handler/couchbase_handler.py (hunk 1 above)
Comment (David00154) [reply mindsdb/integrations/handlers/couchbase_handler/couchbase_handler.py:147]: OK will make the changes

**Review by ZoranPandovski — APPROVED** [review]

**Pull request merged**